linux_pipeline/Jenkinsfile_azure_stress_networking_watcher

```diff
@@ -0,0 +1,54 @@
+#!/usr/bin/env groovy
+
+properties ([
+    pipelineTriggers([cron('H 1 * * *')])
+])
+
+env.WORK_DIR = "/mnt/tmp/watcher-azure-stress-networking"
+env.IMAGE_TAGS = "Ubuntu 18.04-LTS"
+env.PIPELINE_NAME = "pipeline-azure-stress-networking"
```

Comment:
@ader1990 This is about new image monitoring in Azure Market. What makes you pick up 'stress' in pipeline name?
Can you choose something else like 'monitor', 'sanity-check', etc?

Reply:
that was the initial test case, I will change the name to watcher-azure-latest-image

Reply:
Thanks . I like it.